src/main/java/com/elytradev/concrete/block/ExpDropBehaviour.java

```diff
@@ -39,21 +39,26 @@
 /**
  * An interface used to describe the behaviour for exp drops for a {@link ConcreteBlock}.
  */
-public interface ExpDropBehaviour {
+public abstract class ExpDropBehaviour {
```

Comment:
Revert changes to this class. Retrolambda supports static members and default methods.